lib/src/controller.dart

```diff
@@ -101,7 +108,9 @@ class WizardController extends ChangeNotifier {
       return routeNames[index + 1];
     }
 
+    _addAsyncOps(1);
     final name = await onNext() ?? nextRoute();
+    _addAsyncOps(-1);
```

Comment:
it would be nice to have the same for going backward but naming is a bit harder then :)